libraries/AP_Mount/AP_Mount_config.h

```diff
@@ -62,6 +62,10 @@
 #define HAL_MOUNT_XFROBOT_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && HAL_PROGRAM_SIZE_LIMIT_KB > 1024
 #endif
 
+#ifndef HAL_MOUNT_AVT_ENABLED
+#define HAL_MOUNT_AVT_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && HAL_GCS_ENABLED && HAL_PROGRAM_SIZE_LIMIT_KB > 1024
+#endif
+
```

Comment:
```suggestion
```

Reply:
we've renamed the backend to MAVLink so we no longer have the AVT specific definition